packages/customer-erasure/src/main.js

```diff
@@ -0,0 +1,217 @@
+import fetch from 'node-fetch'
+import { createClient } from '@commercetools/sdk-client'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import {
+  createAuthMiddlewareForClientCredentialsFlow,
+  createAuthMiddlewareWithExistingToken,
+} from '@commercetools/sdk-middleware-auth'
+import { createUserAgentMiddleware } from '@commercetools/sdk-middleware-user-agent'
+import flatten from 'lodash.flatten'
+import silentLogger from './utils/silent-logger'
+import pkg from '../package.json'
+
+// todo add flow types
```

Comment:
A bit sad to see this as a "nice-to-have" thing and not a core part of the development.